testsuite/tests/parsing/dropped_attributes.ml

```diff
@@ -0,0 +1,15 @@
+(* TEST
+   ocamlc_byte_exit_status = "2";
+   setup-ocamlc.byte-build-env;
+   ocamlc.byte;
+   check-ocamlc.byte-output;
+*)
+
+(* Attributes [[@annot1]] and [[@annot3]] have not node in the parsetree
+   to attach to; make sure the parser doesn't drop them (which it used to do). *)
+
+module _ : sig
+  val foo : (module T [@annot1] with type t = 'a [@annot2]) -> unit
```

Comment:
Are we testing anything with `annot2` and `annot4`?

Reply:
No, this was part of a compound bug where ocamlformat was dropping those two attributes. I'll remove them once.

Reply:
Removed the unrelated attributes.